src/renderer/features/multisig-operations/model/feature.ts

```diff
@@ -42,35 +43,39 @@ const $input = combine(
   ({ apis, chains, wallet }) => {
     if (nullable(wallet) || !walletUtils.isMultisig(wallet)) return null;
 
-    const input = [];
+    const account = wallet.accounts.find(accountUtils.isAnyMultisigAccount);
+    if (nullable(account)) return null;
 
-    for (const account of wallet.accounts) {
-      let availableChains;
-      let accountId: AccountId;
-      if (accountUtils.isFlexibleMultisigAccount(account)) {
-        const chain = chains[account.chainId];
-        availableChains = chain ? [chain] : [];
-        accountId = account.multisigAccountId;
-      } else {
-        availableChains = Object.values(chains).filter(chain => networkUtils.isMultisigSupported(chain.options));
-        accountId = account.accountId;
-      }
+    let availableChains;
+    let accountId: AccountId;
+
+    if (accountUtils.isFlexibleMultisigAccount(account)) {
+      const chain = chains[account.chainId];
+      availableChains = chain ? [chain] : [];
+      accountId = account.multisigAccountId;
+    } else {
+      availableChains = Object.values(chains).filter(chain => {
+        return networkUtils.isMultisigSupported(chain.options) && accountService.isCryptoMatch(account, chain);
+      });
+      accountId = account.accountId;
+    }
```

Comment:
This really looks like we are putting responsibility of checking account-related restrictions on the wrong entity
Why is this not a part of `accountSdk`?